qiskit_ibm_runtime/options/noise_learner_options.py

```diff
@@ -0,0 +1,85 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""NoiseLearner options."""
+
+from typing import List, Union
+
+from pydantic import ValidationInfo, field_validator
+
+from .utils import Unset, UnsetType
+from .twirling_options import TwirlingStrategyType
+from .options import OptionsV2
+from .utils import primitive_dataclass, make_constraint_validator, skip_unset_validation
+
+
+@primitive_dataclass
+class NoiseLearnerOptions(OptionsV2):
+    """Options for :class:`.NoiseLearner`.
```

Comment:
Since this is a new class, we can start using attribute docstring instead of argument docstring (see https://github.com/Qiskit/qiskit-ibm-runtime/issues/1762) to make the API reference look nicer.